bazel/envoy_build_system.bzl

```diff
@@ -56,6 +56,8 @@ def envoy_cc_library(name,
 def envoy_cc_binary(name,
                     srcs = [],
                     data = [],
+                    linkstatic = 0,
+                    visibility = None,
```

Comment:
`linkstatic` is used now to build both dynamic and static binaries in `source/exe/BUILD`. `visibility` is unused, I was using it while playing around with Istio, but it doesn't hurt to have it.